api/v1alpha1/clusterextension_types.go

```diff
@@ -111,13 +121,15 @@ func init() {
 	conditionsets.ConditionReasons = append(conditionsets.ConditionReasons,
 		ReasonInstallationSucceeded,
 		ReasonResolutionFailed,
-		ReasonResolutionUnknown,
-		ReasonBundleLookupFailed,
 		ReasonInstallationFailed,
-		ReasonInstallationStatusUnknown,
-		ReasonInvalidSpec,
 		ReasonSuccess,
 		ReasonDeprecated,
+		ReasonUpgradeFailed,
+		ReasonBundleLoadFailed,
+		ReasonErrorGettingClient,
+		// TODO: this reason is not being used in the reconciler, it will be removed
+		// when we fix the tests. Avoiding removal here, to reduce diffs.
+		ReasonInstallationStatusUnknown,
```

Comment:
Now that there has been a commit to fix the tests can this reason be removed?

Reply:
We'll remove this and the TODO.
OWNER: @varshaprasad96

Reply:
Removed TODO in https://github.com/operator-framework/operator-controller/pull/887. If its not needed after fixing unit-test, we will address in #874